qlib/contrib/data/highfreq_handler.py

```diff
@@ -110,6 +110,92 @@ def get_normalized_price_feature(price_field, shift=0):
         return fields, names
 
 
+class HighFreqGeneralHandler(HighFreqHandler):
```

Comment:
Should this class inherit from  DataHandlerLP directly?

Both `class HighFreqHandler` and `class HighFreqGeneralHandler` are special cases of the high frequent handlers.
We don't have a base class for high frequent handlers. So it should inherit from DataHandlerLP directly